docs/delegation_design_spec/delegation_design_spec.tex

```diff
@@ -43,15 +42,12 @@
 \begin{document}
 
 \title{Design Specification for Delegation and Incentives in Cardano \\
-       {\small (Version 1.2, git revision \gitAbbrevHash\gitDirty)} \\
        {\large \sc An IOHK technical report}}
 
 \author{Philipp Kant   \\ {\small \texttt{philipp.kant@iohk.io}} \\
    \and Lars Br\"unjes \\ {\small \texttt{lars.bruenjes@iohk.io}} \\
    \and Duncan Coutts  \\ {\small \texttt{duncan@well-typed.com}} \\
                           {\small \texttt{duncan.coutts@iohk.io}}}
-\date{January, 2019}
```

Comment:
Since we are versioning with dates now instead of git revisions, I just wanted to check with you @nc6 that you like auto-generating the dates.

Reply:
Seems fine with me, as it apparently woks on hydra!